src/commcare_cloud/terraform/modules/ga_alb_waf/main.tf

```diff
@@ -52,6 +52,9 @@ resource "aws_wafv2_ip_set" "permanent_block" {
   ip_address_version = "IPV4"
   description        = "Manually Added IPs for denial"
   addresses          = ["195.54.160.21/32", "64.39.99.208/32"]
+  lifecycle {
```

Comment:
Just so I understand, we still hard code/reference addresses listed here, but will ignore changes made to addresses?

Reply:
Yeah sorry this would be less dissonant if I also replaced the addresses value above with an empty list. I'll do that